docs/configuration.rst

```diff
@@ -79,6 +79,131 @@ Rally will ask you a few more things in the advanced setup:
 * **Name for this benchmark environment** (only for metrics store type ``elasticsearch``): You can use the same metrics store for multiple environments (e.g. local, continuous integration etc.) so you can separate metrics from different environments by choosing a different name.
 * whether or not Rally should keep the Elasticsearch benchmark candidate installation including all data by default. This will use lots of disk space so you should wipe ``~/.rally/benchmarks/races`` regularly.
 
+Configuration File Reference
+----------------------------
+
+Rally stores its configuration in the file ``~/.rally/rally.ini``. It contains of the following sections.
```

Comment:
`s/It contains of the following sections./It comprises the following sections:`

Reply:
addressed in https://github.com/elastic/rally/pull/1137/commits/415c198c5c289f6533ae27e0895e174cd0f4bbd8